Documentation~/Blender.md

```diff
@@ -27,3 +27,42 @@ The plugin will send the following properties directly to Unity:
 > Other properties will require **Bake Modifiers** enabled to get consistent look between Blender and Unity.
 > However, this will also cause the loss of mesh properties in Unity as result of the baking.
 
+---
+
+## Bidirectional Sync
+
+Turn on **Auto-Sync** in the plugin panel to apply property changes in Unity back to Blender.  
+This feature is intended for changing Blender's procedural generation parameters from inside Unity, and 
+requires Blender 3.0 or higher.
+
+
+### Editable Properties in Unity
+
+![](images/MeshSyncServerLiveEditProperties.png)
+
+We can set up custom properties in Blender, from which the plugin will synchronize these properties with 
```

Comment:
Maybe change this to:
MeshSync synchronizes custom properties on objects and geometry node inputs from Blender with the MeshSyncServer **GameObject** in Unity. 
If needed, a MeshSyncServerLiveEditProperties component is created to store, display and edit these properties.